test/sourcemaps.js

```diff
@@ -141,7 +140,7 @@ suite('Source Maps', function() {
           '../fixtures/test-tree/some.js',
           '../fixtures/test-tree/text.txt',
           '../fixtures/test-tree/amd.js',
-          '../fixtures/test-tree/component.jsx',
+          '../fixtures/test-tree/unknown', // component.jsx
```

Comment:
Yes, I think previously the sourcemap contained `component.jsx → unknown` (but traceur for some reason ignored the latter), but now babel ignores the intermediate file.